src/main/resources/application.properties

```diff
@@ -32,10 +32,12 @@ grpc.server.enable-keep-alive=true
 grpc.server.keep-alive-time=30s
 grpc.server.keep-alive-timeout=15s
 grpc.server.permit-keep-alive-without-calls=true
-grpc.server.security.enabled=false
+grpc.server.security.enabled=true
+grpc.server.security.certificate-chain=classpath:certificates/server.crt
+grpc.server.security.private-key=classpath:certificates/server.key
```

Comment:
Read the issue more carefully -> "`grpc.server.security.privateKey` should be passed as a string"
https://dev.azure.com/ampnetio/AMPnet%20Crowdfunding/_workitems/edit/489